lib/common/operations.c

```diff
@@ -362,14 +362,14 @@ decode_transition_key(const char *key, char **uuid, int *transition_id, int *act
     return TRUE;
 }
 
-#define CRM_META_LEN (sizeof(CRM_META) - 1)
+// String length of CRM_META"_"
+#define CRM_META_LEN sizeof(CRM_META)
```

Comment:
instead of the comment the following should evaluate a compile time - right
#define CRM_META_LEN (sizeof(CRM_META"_") - 1)

Reply:
Yes, and it's only used once, so might as well drop the constant. I'll add a separate commit